native/cocos/editor-support/spine-wasm/spine-skeleton-instance.cpp

```diff
@@ -34,7 +33,6 @@ static void trackEntryCallback(AnimationState *state, EventType type, TrackEntry
 
 SpineSkeletonInstance::SpineSkeletonInstance() {
     _model = new SpineModel();
-    SpineSkeletonSystem::addSpineInstance(this);
```

Comment:
Why delete SpineSkeletonSystem ?

Reply:
Its no need an additional helper class to release the spineSkeletonInstance object.